### PR TITLE
Add EaglePointSoftware.PinnacleSeriesManagementUtility 2023.2.1.101

### PR DESCRIPTION
**Pull request by JasonCook599**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98092)

### CODE REVIEW

File: manifests/e/EaglePointSoftware/PinnacleSeriesManagementUtility/2023.2.1.101/EaglePointSoftware.PinnacleSeriesManagementUtility.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+PackageIdentifier: EaglePointSoftware.PinnacleSeriesManagementUtility
+PackageVersion: 2023.2.1.101
+PackageLocale: en-US
+Publisher: Eagle Point Software
+PublisherUrl: https://www.eaglepoint.com/
+PublisherSupportUrl: https://support.eaglepoint.com/
+PrivacyUrl: https://www.eaglepoint.com/privacy-policy/
+Author: Eagle Point Software Corporation
+PackageName: Pinnacle Series Management Utility
```
Comment (Trenly) [inline manifests/e/EaglePointSoftware/PinnacleSeriesManagementUtility/2023.2.1.101/EaglePointSoftware.PinnacleSeriesManagementUtility.locale.en-US.yaml:12]:
```suggestion
PackageName: Pinnacle Series
```
This should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Might we change the PackageIdentifier to something like `EaglePoint.PinnacleSeries` to make it more concise?

File: manifests/e/EaglePointSoftware/PinnacleSeriesManagementUtility/2023.2.1.101/EaglePointSoftware.PinnacleSeriesManagementUtility.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+PublisherUrl: https://www.eaglepoint.com/
+PublisherSupportUrl: https://support.eaglepoint.com/
+PrivacyUrl: https://www.eaglepoint.com/privacy-policy/
+Author: Eagle Point Software Corporation
+PackageName: Pinnacle Series
+License: Proprietary
+Copyright: Copyright © 2007-2018, by Eagle Point Software Corporation. All rights reserved.
+ShortDescription: user administration, platform branding, content customization, course enrollments, and reporting
+Description: Platform Administrators and Managers require a "thin client application" (Windows Based) to be installed onto their workstations. The Pinnacle Series Management Utility is required for user administration, platform branding, content customization, course enrollments, and reporting.
+Moniker: Pinnacle
```
Comment (mdanish-kh) [inline manifests/e/EaglePointSoftware/PinnacleSeriesManagementUtility/2023.2.1.101/EaglePointSoftware.PinnacleSeriesManagementUtility.locale.en-US.yaml:17]:
```suggestion
Moniker: pinnacle
```
A moniker is a special type of tag. It should be written in all lowercase.

**Comment by stephengillie** [comment]
No recent activity.

**Pull request closed without merging**